apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.spec.tsx

```diff
@@ -60,7 +60,7 @@ describe('Confirmation route', () => {
         name: /Staðfesta og hefja undirritun/i,
       }),
     ).not.toBeDisabled()
-  })
+  }, 10000)
```

Comment:
We've set a global timeout for tests at 60 seconds, so this is not needed
```suggestion
  })
```